aws/elastic-beanstalk/cloudformation/beanstalk.yaml.tmpl

```diff
@@ -0,0 +1,187 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Elasticbeanstalk test template
+Parameters:
+  BeanstalkService:
+    Type: String
+  Ec2Service:
+    Type: String
+  Partition:
+    Type: String
+  SolutionStackName:
+    Type: String
+
+Resources:
+
+  # ----------------------------------------------------------------------------
+  Application:
+    Type: AWS::ElasticBeanstalk::Application
+    Properties:
+      ApplicationVersions:
+        - Description: Version 1.0
+          SourceBundle:
+            S3Bucket: {{.S3BucketName}}
```

Comment:
can we pass this as Parameter from the XLD instead of hardcoding here

Reply:
Turned into parameter